### PR TITLE
PORTALS-2721, PORTALS-2718, PORTALS-2733

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Columns icon:
Screen Shot 2023-07-31 at 11 24 07 AM

**Comment by jay-hodgson** [comment]
Left padding on .checkbox ok in different contexts:
JSON Schema form:
Screen Shot 2023-07-31 at 12 01 35 PM

Facets (helps to line up with radio buttons!):
Screen Shot 2023-07-31 at 12 02 03 PM

Row selection:
Screen Shot 2023-07-31 at 12 02 16 PM

File: apps/portals/src/configurations/elportal/headerConfig.tsx [hunk 1]
```diff
@@ -14,8 +14,9 @@ const homePageHeader: HomePageHeaderConfig = {
         target="_blank"
         sx={{
           color: 'white',
-          '&:hover': { color: 'white' },
+          '&:hover': { color: 'white', textDecorationColor: 'white' },
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/elportal/headerConfig.tsx:17]: https://sagebionetworks.jira.com/browse/PORTALS-2718

File: apps/portals/src/configurations/nf/headerConfig.tsx [hunk 2]
```diff
@@ -15,8 +15,9 @@ const homePageHeader: HomePageHeaderConfig = {
         target="_blank"
         sx={{
           color: 'white',
-          '&:hover': { color: 'white' },
+          '&:hover': { color: 'white', textDecorationColor: 'white' },
```
Comment (jay-hodgson) [inline apps/portals/src/configurations/nf/headerConfig.tsx:18]: https://sagebionetworks.jira.com/browse/PORTALS-2718

File: packages/synapse-react-client/src/assets/themed_icons/Columns.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ export const Columns = () => (
       y="0.5"
       width="4.66667"
       height="16"
-      stroke="white"
+      stroke="none"
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/assets/themed_icons/Columns.tsx:19]: https://sagebionetworks.jira.com/browse/PORTALS-2721

File: packages/synapse-react-client/src/assets/themed_icons/ColumnsDark.tsx [hunk 4]
```diff
@@ -21,8 +21,8 @@ export const ColumnsDark = () => (
       y="0.5"
       width="4.66667"
       height="16"
-      stroke="#ffffff"
-      fill="#ffffff"
+      stroke="none"
+      fill="none"
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/assets/themed_icons/ColumnsDark.tsx:25]: https://sagebionetworks.jira.com/browse/PORTALS-2721

File: packages/synapse-react-client/src/template_style/_form.scss [hunk 5]
```diff
@@ -16,7 +16,7 @@
 
 .checkbox,
 .radio {
-  padding-left: 0px;
+  padding-left: 6px;
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/template_style/_form.scss:19]: https://sagebionetworks.jira.com/browse/PORTALS-2732 (not the complete fix, so leaving this bug as open)

**Review by nickgros — APPROVED** [review]

File: packages/synapse-react-client/src/template_style/_form.scss (hunk 5 above)
Comment (nickgros) [reply packages/synapse-react-client/src/template_style/_form.scss:19]: Would it make more sense to add margin to the table cell renderer?

Comment (jay-hodgson) [reply packages/synapse-react-client/src/template_style/_form.scss:19]: Recommend we do both.  
In general, I think the checkbox should not bump against the left side edge (it already seems to have right padding).  Adam wants this to be wider in the table cell context, so PORTALS-2732 is not fixed by this change (but it is improved).

**Pull request merged**